ppg-server/ppg-server.spec

```diff
@@ -0,0 +1,30 @@
+%global sname   percona-ppg-server
+%global pgmajorversion 14
+%global version 4
+
+Summary:        Base selection of PostgreSQL%{pgmajorversion} components
```

Comment:
This description isn't accurate. It's not a PG community package.

Reply:
Fixed that